v1/fleet-local/control/booster-control@.service

```
@@ -0,0 +1,40 @@
+[Unit]
+Description=Booster Control @ %i
+Requires=docker.service
+After=docker.service
+
+[Service]
+User=core
+Restart=on-failure
+Environment="IMAGE=etcdctl get /images/booster"
+
+ExecStartPre=/usr/bin/sh -c "/usr/bin/docker pull $($IMAGE)"
+
+# mkdir so `etcdctl ls /booster/nodes` below doesn't fail
+ExecStartPre=-/usr/bin/sh -c "etcdctl mkdir /booster/nodes"
```

It does fail if the directory already exists. `ExecStartPre=-` ignores the error